### PR TITLE
Update SECURITY.md

### LINKED ISSUE #1509: [DOCS] UPDATE README,SECURITY,AND CONTRIBUTING PAGE
### Issue Description

There are a lot of grammatical errors in the README, SECURITY,AND CONTRIBUTING PAGE

### Additional Context

_No response_

### Screenshots or Examples (if applicable)

_No response_

### Proposed Solution (optional)

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by abhisar-yadav**
Improved the framing of statements 
## Fixes 
Closes #1509 
Relates to #1553 & #1554

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 ## Introduction
 
-This document outlines the security policy for the [Linkshub](https://linkshub.vercel.app/) open source project that provides links to websites for courses across the internet. The policy is intended to provide guidelines and procedures for reporting, triaging, and addressing security vulnerabilities in the project.
+This document outlines the security policy for the [Linkshub](https://linkshub.vercel.app/) open-source project that provides links to websites for courses across the internet. The policy is intended to provide guidelines and procedures for reporting, triaging, and addressing security vulnerabilities in the project.
```
Comment (CBID2) [inline SECURITY.md:5]:
```suggestion
This document outlines the security policy for the [Linkshub](https://linkshub.vercel.app/) open source project that provides links to websites for courses across the internet. The policy is intended to provide guidelines and procedures for reporting, triaging, and addressing security vulnerabilities in the project.
```

File: SECURITY.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
 ## Scope
 
-The security policy covers the codebase and documentation of the open source project, as well as the external links to websites for courses.
+The security policy covers the codebase and documentation of the open-source project, as well as the external links to websites for courses.
```
Comment (CBID2) [inline SECURITY.md:9]:
```suggestion
The security policy covers the codebase and documentation of the open source project, as well as the external links to websites for courses.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @abhisar-yadav. I made some suggestions. Note: "open source" is the correct spelling.

**Comment by k-deepak04** [comment]
@abhisar-yadav have you made the changes requested by @CBID2 ?

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**